apps/server/lib/http/routes.ts

```diff
@@ -172,7 +175,7 @@ export const attachRoutes = (
 
 				return response.status(200).json({
 					error: false,
-					data: _.omit(results.data, ['links']),
+					data: _.omit(results.data as any, ['links']),
```

Comment:
There's almost no valid reason for an `as any` cast. Is it really necessary here?